From 6a33a12798b2afeee6fb7bba74d86d628137921e Mon Sep 17 00:00:00 2001
From: Nicholas Marriott <nicholas.marriott@gmail.com>
Date: Thu, 30 Apr 2020 15:20:08 +0100
Subject: [PATCH] Do not remove the automatic-rename option from the global
 set, only from the window (it must stay in the global set or tmux will
 crash). GitHub issue 2188.

--- input.c.orig	2020-04-29 20:01:18 UTC
+++ input.c
@@ -2304,7 +2304,7 @@ input_exit_rename(struct input_ctx *ictx)
 		return;
 
 	if (ictx->input_len == 0) {
-		oe = options_get(wp->window->options, "automatic-rename");
+		oe = options_get_only(wp->window->options, "automatic-rename");
 		if (oe != NULL)
 			options_remove(oe);
 		return;
